docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.        while start &lt; chunk_stop:
</pre><p>Is this code readable? I tried to vectorize it but couldn't figure out "how". If the number of subsequences in each chunk were supposed to be a power of two, I think I would be able to vectorize it. However, according to the paper, the size of the chunk itself should be a power of two. In other words, the number of subsequences is like... <code>2 ** num - m + 1</code> </p><p> </p><p>Since I couldn't find out a cleaner solution, I am going with <code>naive_get_range_damp</code> for now. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>